clouddriver-eureka/clouddriver-eureka.gradle

```diff
@@ -0,0 +1,3 @@
+dependencies {
```

Comment:
What are your thoughts on making this module contain all supported service discovery types (starting with eureka), since there is some interest here at Google to add support for Consul